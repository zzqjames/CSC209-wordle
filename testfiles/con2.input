y 3 ---yy -gy-g pitch
y 4 ---yy -gy-g pitch
c strip
c chmxs