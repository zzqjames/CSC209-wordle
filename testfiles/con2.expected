cannot_be: c h i m p r s t x 
must_be
[0] 
[1] 
[2] 
[3] i t h 
[4] i t 
